bower.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "beard-server",
+  "description": "Application providing REST API over Beard",
+  "directory": "beard_server/static/",
+  "authors": [
+    "Grzegorz Jacenków <grzegorz.jacenkow@cern.ch>"
+  ],
+  "homepage": "https://github.com/inspirehep/beard-server",
+  "dependencies": {
+    "angular": "1.x.x",
```

Comment:
Wouldn't it be better to pin the last part (minor version) in all these libraries? For example, from Angular 1.1 to 1.5 there would be many breaking changes.